pkg/apis/resources.cattle.io/v1/types.go

```diff
@@ -112,6 +112,9 @@ type RestoreSpec struct {
 	Prune                      *bool            `json:"prune"` //prune by default
 	DeleteTimeoutSeconds       int              `json:"deleteTimeoutSeconds,omitempty"`
 	EncryptionConfigSecretName string           `json:"encryptionConfigSecretName,omitempty"`
+
+	// When set to true, controller will ignore any error during restore CRs and CRDs and mark it as done
+	IgnoreError bool `json:"ignoreError,omitempty"`
```

Comment:
Nit, but maybe have it plural since we're ignoring any number of records and not necessarily one error. Obviously, CRD yaml needs to be updated accordingly.
```suggestion
	IgnoreErrors bool `json:"ignoreErrors,omitempty"`
```